test_suite/package/otel_package/test_otel.py

```diff
@@ -69,5 +70,5 @@ def test_otel(self, host):
         console_lib.print_divider("Check for error logs in aws cli logs")
         assert "Invalid user" in self.check_aws_cli_logs(host, self.instance_region)
 
-        if float(host.system_info.release) > 9.4:
+        if version.parse(host.system_info.release) > version.parse('9.4'):
```

Comment:
I think this check is redundant, since the test case is already applicable only to RHEL-9.5 and later versions.

I suggest to remove the check and always run the check for avc denials.

Reply:
ACK